src/main/scala/cromwell/binding/Call.scala

```diff
@@ -1,11 +1,56 @@
 package cromwell.binding
 
 import cromwell.binding.types.WdlType
-import cromwell.binding.values.WdlValue
 import cromwell.parser.AstTools
+import cromwell.parser.WdlParser.{SyntaxError, Terminal, Ast}
+import cromwell.parser.AstTools.EnhancedAstNode
 
 import scala.util.{Success, Try}
 
+object Call {
```

Comment:
Okay I don't think I quite get why all these `apply` methods were added.  My understanding of this is that now instead of `processCall`, there's an `apply()` method on Call that essentially does that.  If that's correct, I think that's a fine refactoring, but did it serve any other purpose that I'm missing?